### ISSUE TITLE
<blaze-template> tag included in rendered output

### ISSUE THREAD

**Issue by borisfeldmann** [Urigo/angular-meteor#756]
Hi there, 

when I use `<blaze-template name="_loginButtons"></blaze-template>` the `<blaze-template>` tag is included in the rendered output, between my `<ul>` and loginButton's `<li>` tag.

Is there a way to prevent this and get a clean `<ul><li>`strutcture?

Here's my package list.

```javascript
accounts-password             1.1.3  Password support for accounts
alanning:roles                1.2.14  Authorization package for Meteor
angular                       1.0.2  Everything you need to use AngularJS in your Meteor app
angularui:angular-ui-router   0.2.15  angular-ui-router (official): Flexible routing with nested views in AngularJS
autopublish                   1.0.4  (For prototyping only) Publish the entire database to all clients
blaze-html-templates          1.0.1  Compile HTML templates into reactive UI with Meteor Blaze
ecmascript                    0.1.5  Compiler plugin that supports ES2015+ in all .js files
es5-shim                      4.1.13  Shims and polyfills to improve ECMAScript 5 support
ian:accounts-ui-bootstrap-3   1.2.81* Bootstrap-styled accounts-ui with multi-language support.
ian:bootstrap-3               3.3.1  HTML, CSS, and JS framework for developing responsive, mobile first projects on the web.
jagi:astronomy                1.0.2* Model layer for Meteor
jagi:astronomy-validators     1.0.3* Validators for Meteor Astronomy
jquery                        1.11.4  Manipulate the DOM using CSS selectors
meteor-base                   1.0.1  Packages that every Meteor app needs
meteortoys:allthings          2.1.0* Insanely Handy Development Tools for Meteor
mobile-experience             1.0.1  Packages for a great mobile user experience
mongo                         1.1.2  Adaptor for using MongoDB and Minimongo over DDP
sanjo:jasmine                 0.20.2  Velocity integration of the Jasmine testing framework
session                       1.1.1  Session variable
standard-minifiers            1.0.1  Standard minifiers used with Meteor apps by default.
tracker                       1.0.9  Dependency tracker to allow reactive callbacks
urigo:angular-blaze-template  0.0.1  Include Blaze templates in your angular-meteor application.
xolvio:cucumber               0.14.10  CucumberJS for Velocity
```

Thanks for your help and your contribution!
Boris

**Comment by Urigo** [comment]
@borisfeldmann that's a a good idea and I think it's pretty easy to do.
We should use the `replace: true` option inside the directive.
It might be a better default behavior actually for the `<blaze-template>` directive.
Because this is relatively simple, I would love to get a pull request, here is the file that needs to be changed:
https://github.com/Urigo/angular-blaze-template/blob/master/angular-blaze-template.js

I wonder if we can also make it optional and changeable with parameter, by I'm not sure it's possible

**Comment by borisfeldmann** [comment]
Hello Uri,

I’d really like to fix this and send you a pull request, but I do not know how to do this. I tried the „replace: true“ option, but it seems to be deprecated: http://stackoverflow.com/questions/24194972/why-is-replace-deprecated-in-angularjs and it is not documented in the latest developer guide https://docs.angularjs.org/guide/directive.

Sorry, Boris


Am 14.10.2015 um 06:46 schrieb Uri Goldshtein <notifications@github.com<mailto:notifications@github.com>>:


@borisfeldmann<https://github.com/borisfeldmann> that's a a good idea and I think it's pretty easy to do.
We should use the replace: true option inside the directive.
It might be a better default behavior actually for the <blaze-template> directive.
Because this is relatively simple, I would love to get a pull request, here is the file that needs to be changed:
https://github.com/Urigo/angular-blaze-template/blob/master/angular-blaze-template.js

I wonder if we can also make it optional and changeable with parameter, by I'm not sure it's possible

—
Reply to this email directly or view it on GitHub<https://github.com/Urigo/angular-meteor/issues/756#issuecomment-147933948>.


---
Werkdigital GmbH
Boris Feldmann, Dipl.-Ing. (FH)
Geschäftsführender Gesellschafter

---
Fon: +49 2721 94925 20
Mob: +49 151 23042508
Fax: +49 2721 94925 90

---
Agathastraße 63
57368 Lennestadt
Germany

---
www.werkdigital.de

---
Sitz der Gesellschaft: Lennestadt
Amtsgericht Siegen / HRB 9350
Geschäftsführung: Dipl.-Inform. Dennis Conze / Dipl.-Ing. (FH) Boris Feldmann

**Comment by dotansimha** [comment]
Hi @borisfeldmann @Urigo 
Just fixed it:
https://github.com/Urigo/angular-blaze-template/pull/1

**Comment by borisfeldmann** [comment]
Great, thanks!

Von meinem iPhone gesendet

Am 18.10.2015 um 21:11 schrieb Dotan Simha <notifications@github.com<mailto:notifications@github.com>>:


Hi @borisfeldmann<https://github.com/borisfeldmann> @Urigo<https://github.com/Urigo>
Just fixed it:
Urigo/angular-blaze-template#1<https://github.com/Urigo/angular-blaze-template/pull/1>

-
Reply to this email directly or view it on GitHub<https://github.com/Urigo/angular-meteor/issues/756#issuecomment-149039775>.

---
Werkdigital GmbH
Boris Feldmann, Dipl.-Ing. (FH)
Gesch?ftsf?hrender Gesellschafter

---
Fon: +49 2721 94925 20
Mob: +49 151 23042508
Fax: +49 2721 94925 90

---
Agathastra?e 63
57368 Lennestadt
Germany

---
www.werkdigital.de

---
Sitz der Gesellschaft: Lennestadt
Amtsgericht Siegen / HRB 9350
Gesch?ftsf?hrung: Dipl.-Inform. Dennis Conze / Dipl.-Ing. (FH) Boris Feldmann

**Comment by Urigo** [comment]
Will be release on the next major version

**Comment by borisfeldmann** [comment]
Great, thanks!

Am 20.10.2015 um 06:50 schrieb Uri Goldshtein <notifications@github.com<mailto:notifications@github.com>>:


Will be release on the next major version

—
Reply to this email directly or view it on GitHub<https://github.com/Urigo/angular-meteor/issues/756#issuecomment-149432729>.


---
Werkdigital GmbH
Boris Feldmann, Dipl.-Ing. (FH)
Geschäftsführender Gesellschafter

---
Fon: +49 2721 94925 20
Mob: +49 151 23042508
Fax: +49 2721 94925 90

---
Agathastraße 63
57368 Lennestadt
Germany

---
www.werkdigital.de

---
Sitz der Gesellschaft: Lennestadt
Amtsgericht Siegen / HRB 9350
Geschäftsführung: Dipl.-Inform. Dennis Conze / Dipl.-Ing. (FH) Boris Feldmann

**Comment by Urigo** [comment]
Released in version 0.1.0
